plugins-bundled/internal/marketplace-app/src/types.ts

```diff
@@ -0,0 +1,79 @@
+export interface MarketplaceAppSettings {
+  includeEnterprise?: boolean;
+}
```

Comment:
I think this is missing `includeUnsigned`
```suggestion
export interface MarketplaceAppSettings {
  includeUnsigned?: boolean;
  includeEnterprise?: boolean;
}
```

Reply:
It needs removing unfortunately. I think I've fixed those errors / warnings now.